sdk/identity/azure-identity/azure/identity/_internal/decorators.py

```diff
@@ -22,12 +22,17 @@ def wrapper(*args, **kwargs):
         try:
             token = fn(*args, **kwargs)
             _LOGGER.log(
-                logging.DEBUG if within_credential_chain.get() else logging.INFO, "%s succeeded", fn.__qualname__
+                logging.DEBUG if within_credential_chain.get() else logging.INFO,
+                "%s succeeded",
+                fn.__qualname__,
             )
             if _LOGGER.isEnabledFor(logging.DEBUG):
                 try:
-                    base64_meta_data = token.token.split(".")[1].encode("utf-8") + b"=="
-                    json_bytes = base64.decodebytes(base64_meta_data)
+                    base64_meta_data = token.token.split(".")[1]
+                    padding_needed = -(len(base64_meta_data) % 4)
```

Comment:
No need for the encapsulating parens (this would give a negative padding_needed value):
```suggestion
                    padding_needed = -len(base64_meta_data) % 4
```